components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/backchannellogout/DefaultLogoutTokenBuilder.java

```diff
@@ -161,15 +161,15 @@ private JWTClaimsSet buildJwtToken(OIDCSessionState sessionState, String tenantD
         long currentTimeInMillis = Calendar.getInstance().getTimeInMillis();
         Date iat = new Date(currentTimeInMillis);
         String sid = getSidClaim(sessionState);
-        JSONObject event = new JSONObject().put("http://schemas.openidnet/event/backchannel-logout",
+        JSONObject event = new JSONObject().appendField("http://schemas.openidnet/event/backchannel-logout",
                 new JSONObject());
 
         JWTClaimsSet.Builder jwtClaimsSetBuilder = new JWTClaimsSet.Builder();
         jwtClaimsSetBuilder.subject(sub);
         jwtClaimsSetBuilder.issuer(iss);
         jwtClaimsSetBuilder.audience(audience);
         jwtClaimsSetBuilder.claim("jti", jti);
-        jwtClaimsSetBuilder.claim("event", event);
+        jwtClaimsSetBuilder.claim("events", event);
```

Comment:
@piraveena was the previous behaviour wrong according the spec?

Reply:
previouslt we had it as **event**. But spec says it should be **events**